src/components/MultiHopTrade/types.ts

```diff
@@ -0,0 +1,21 @@
+export enum MultiHopExecutionStatus {
+  TradeFailed = 0,
+  Hop1AwaitingApprovalConfirmation = 1,
+  Hop1AwaitingApprovalExecution = 2,
+  Hop1AwaitingTradeConfirmation = 3,
+  Hop1AwaitingTradeExecution = 4,
+  Hop2AwaitingApprovalConfirmation = 5,
+  Hop2AwaitingApprovalExecution = 6,
+  Hop2AwaitingTradeConfirmation = 7,
+  Hop2AwaitingTradeExecution = 8,
+  TradeComplete = 9,
+  TradeCancelled = 10,
```

Comment:
This is fit for purpose but later we'll need to be conscious of the idea of "cancelled" and "error" as not being included in the ordered set of operations when determining `next()` in our execution flow